condor/job_mapper

```diff
@@ -70,12 +70,13 @@ def build_matrix(energy, chosen_sample = None, chosen_mode = None, splitTTJet =
         if not chosen_sample in samples:
             sys.exit("ERROR Trying to run unknown sample '%s'" % options.sample)
         for mode in analysis_modes_:
-            if 'TTJets' in chosen_sample:
+            if 'TTJets' in chosen_sample or 'SingleMuon' in chosen_sample:
```

Comment:
I assume this change is done so we do not run analysis modes other than 'central' on data (in addition to the TTJet systematics samples). In that case, can you please add 'SingleElectron'?
```python
if any([s in chosen_sample for s in ['TTJet', 'SingleMuon', 'SingleElectron']]):
```